packager/media/formats/mp2t/adts_header.cc

```diff
@@ -89,6 +88,13 @@ size_t AdtsHeader::GetFrameSize() const {
   return frame_size_;
 }
 
+size_t AdtsHeader::GetFrameSizeWithoutParsing(const uint8_t* data,
+                                              size_t num_bytes) const {
+  DCHECK_GT(num_bytes, 5)
```

Comment:
Missing ";" in the end. 

It causing building failures: https://ci.appveyor.com/project/shaka/shaka-packager/builds/27774035

Reply:
Oi! My lack of sleep is showing. :smile:

Reply:
Also, I have separate folders for the "official" shaka build, and my fork (which lacks the build-related files). I should've taken the time to overlay my fork on top of the folder with the build files.